jobs/env/ci-cri-containerd-e2e-gce.env

```diff
@@ -0,0 +1,20 @@
+### job-env
+# This list should match the list in kubernetes-pull-build-test-e2e-gce.
+NUM_NODES=4
+# For now explicitly test etcd v2 mode in this suite.
+STORAGE_BACKEND=etcd2
+TEST_ETCD_IMAGE=2.2.1
+TEST_ETCD_VERSION=2.2.1
+
+# Enable the PodSecurityPolicy in tests.
+ENABLE_POD_SECURITY_POLICY=true
+
+# Envs for cri-containerd.
+KUBE_MASTER_EXTRA_METADATA="user-data=${GOPATH}/src/github.com/kubernetes-incubator/cri-containerd/test/e2e/master.yaml,cri-containerd-configure-sh=${GOPATH}/src/github.com/kubernetes-incubator/cri-containerd/test/configure.sh"
```

Comment:
we don't allow `$Var` in env file, it's not gonna expand. Maybe just use `/go/src/...` here?

Reply:
Done. Changed to reply on cwd.